examples/partition_example/partition_example.py

```diff
@@ -33,7 +33,7 @@ def relativedelta(*args, **kwargs):
     metadata={"partition_expr": "LastModifiedDate"},
```

Comment:
This file will not get merged. It's an example of what some of these apis will look like in use